09-redux/classworks/Volodkova_Anna/src/pages/Routes.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {Switch, Route, Redirect} from 'react-router-dom';
+
+import useRedditApi from 'hooks/useRedditApi';
+
+import Login from 'pages/login';
+import Callback from 'pages/callback';
+import Home from 'pages/home';
+
+function Routes({reactSubreddit, reactAbout, reactSubreddits, handle}) {
+  const [, , isLoggedIn] = useRedditApi();
+  console.log(isLoggedIn);
+  if (isLoggedIn) {
+    return (
+      <Switch>
+        <Route path="/" exact render={() => <Home reactSubreddit={reactSubreddit} reactAbout={reactAbout}
```

Comment:
no, no, no, no, no
if `Home` needs `reactSubreddit` it should be taken from Redux store, not passed by props